Backend/src/modules/account/dto/failure.rs

```diff
@@ -36,6 +37,7 @@ impl Responder<'static> for Failure {
                 Status::new(523, "PwnedPassword")
             },
             Failure::PasswordTooShort => Status::new(524, "PasswordTooShort"),
+            Failure::InvalidPasswordCharacters => Status::new(422, "InvalidPasswordCharacters"),
```

Comment:
Four points, the last two are just general design questions: 
1. Doing this change was actually not as difficult as I would have thought, I might have to take back some things I said 😄 
2. Not sure if I did all required changes for the backend. There is still frontend work left to be done, so the user knows what's going on. 
3. I used the StatusCode `422 Unprocessable Entity` for this error case. I am pretty confused why every other failure case returns a 5xx error when most of these are caused by the user making bad requests. In my opinion those should return 4xx. WDYT?
4. I am not the biggest fan of this huge `Failure` enum which can be returned by pretty much every endpoint. The problem is that the endpoint for updating a password can return different errors than the errors that can happen during account deletion. I personally prefer every endpoint to have its own specific failure enum. WDYT?

Reply:
1. :D I hope you will learn to love this language!
2. I will check that later :>
3. Its an convention I used, as many 4xx are handled by the browser already and I wanted to return sth + semantically signal that the process is not ok. Many of the 4xx already have a meaning other than what I returned. Plus most of them are not really a client failure. But one could argue about that ^^ Its a design decision that may or may not have been good.
4. Me neither. I changed that for most of the other modules. Each module now has its own failure type. I just need to do some final refactoring. But as this is used in the Account panel and the "Failure" is still mostly associated to that, its fine for now. But I agree that it should get a more precise name :>

Reply:
3. I think it is fine to define custom status codes, but for example the `PasswordTooShort` for which you return 524 is definitely an error of the user rather than the server, so it should be a 4xx error. The reason why I think it is important to make a clear differentiation between 4xx and 5xx is so that one can define a simple metric or alarm showing all 5xx errors to see if the server is doing well.